src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -255,19 +253,23 @@ Capabilities with `CT≠0` are sealed against modification and cannot be derefer
 Instructions that operate on capabilities will produce a result with a cleared {ctag} if the source capability is sealed and the operation would alter its address, bounds, or permissions.
 Extensions that augment capability metadata must describe their interaction(s) with sealed capabilities.
 
-When a sealed capability is used as a function pointer it is known as a <<sentry_cap>>.
-
-[#sentry_cap,reftext="sentry capability"]
-==== Sentry Capabilities
-Sentries describe secure function entry points.
-They have a <<sec_cap_type>> of 1.
-
-They are used as the target of <<JALR_CHERI>>, which must have a zero offset for the use of a sentry to be valid.
-
-Sentry capabilities can establish a form of control-flow integrity between mutually distrusting code.
-<<JALR_CHERI>> with zero offset automatically unseals a sentry target capability and installs it in the <<pcc,program counter capability>>.
-
-<<JALR_CHERI>> also seals the return register, so the callee can return to the caller but cannot access the memory pointed at by the return register, and cannot return elsewhere.
+Given a capability with `CT=0`,
+deriving a capability with `CT≠0` is termed "sealing"
+(or "sealing at ``CT``" when a particular output ``CT`` is meant).
+Dually,
+deriving a `CT=0` capability from a `CT≠0` capability is termed "unsealing"
+(or "unsealing at ``CT``" when a particular input ``CT`` is meant).
+In general, each of these actions may require *authority*
+to operate at the non-zero type;
+extensions will specify how software expresses this authority
+for types not defined above.
+
+[#sec_cap_type_ambient]
+Ambient sealing type::
+Some capability types are said to be "ambiently available" (or just "ambient")
+if they require no particular authority to use to seal a capability.
```

Comment:
```suggestion
if they don't require a specific permission to be granted to seal a capability.
```

I find "no particular authority" quite vague - I think we're saying that there's no specific sealing permission required.